xml/vt_best_practices.xml

```diff
@@ -479,7 +479,12 @@ always madvise [never]</screen>
      consumption on the &vmhost; by sharing data &vmguest;s have in common. The
      KSM daemon <systemitem class="daemon">ksmd</systemitem> periodically scans
      user memory looking for pages of identical content which can be replaced
-     by a single write-protected page. To enable KSM, run:
+     by a single write-protected page. To enable KSM, start the relative service:
+    </para>
+<screen>&prompt.user;&sudo; systemctl enable --now ksm.service</screen>
+    <para>
+     Which will be available if the package <package>qemu-ksm</package>
+     is installed. Alternatively, it can also be started by doing:
```

Comment:
I suggest replacing "doing" with "running the following command:" It is better to be explicit.

Reply:
Ok, done (although slightly different, as I did use the sentence "run the following command" above)

I will update the pull request